src/common/common/clipboard.h

```diff
@@ -72,8 +72,10 @@ typedef struct guac_common_clipboard {
 
 /**
  * Creates a new clipboard.
+ * 
+ * @param buffer_size The buffer size in bytes.
```

Comment:
The style for `@param` should be with the description of the parameter starting on the next line, indented by one level. For example:

https://github.com/apache/guacamole-server/blob/8e75eea9ce7e4b672f55bfb4068b927255156dc0/src/libguac/guacamole/client.h#L445-L455

You'll see some older documentation using the older style, but we are moving to the new style for any new/modified code.

Reply:
Fixed the formatting in the file entirely.